src/pages/Explore/index.tsx

```diff
@@ -1,9 +1,13 @@
 import { Trans } from '@lingui/macro'
-import { BrowserEvent, InterfacePageName, SharedEventName } from '@uniswap/analytics-events'
+import { BrowserEvent, InterfaceElementName, InterfacePageName, SharedEventName } from '@uniswap/analytics-events'
 import { TraceEvent } from 'analytics'
 import { Trace } from 'analytics'
 import { AutoRow } from 'components/Row'
-import { MAX_WIDTH_MEDIA_BREAKPOINT, MEDIUM_MEDIA_BREAKPOINT } from 'components/Tokens/constants'
+import {
+  LARGE_MEDIA_BREAKPOINT,
+  MAX_WIDTH_MEDIA_BREAKPOINT,
+  MEDIUM_MEDIA_BREAKPOINT,
+} from 'components/Tokens/constants'
```

Comment:
non-blocking nit: but it would be nice to deprecate uses of these non-global theme breakpoints and not introduce new uses.